vm_supervisor/conf.py

```diff
@@ -192,7 +192,7 @@ def display(self) -> str:
         annotations = self.__annotations__.copy()
 
         for attr in annotations.keys():
-            if attr in self.SENSITIVE_FIELDS:
+            if getattr(self, attr) and attr in self.SENSITIVE_FIELDS:
```

Comment:
I'm not sure I get the point here. Isn't the point of redacting the attributes that users should have no information whatsoever on the value, even if it is empty?